src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -477,6 +477,12 @@ volumeClaimTemplates:
       name: {{ . }}
       key: {{ $.Values.database.credentials.passwordSecretKey }}
 {{ end }}
+{{- if .Values.database.type }}
```

Comment:
Interesting finding - with helm 3.7 (configured in GH actions) `{{- if and (.Values.database.type) (contains "oracle" .Values.database.type ) }}` does not work when `database.type` isn't set:

```
Error: template: confluence/templates/statefulset.yaml:129:16: executing "confluence/templates/statefulset.yaml" at <include "confluence.databaseEnvVars" .>: error calling include: template: confluence/templates/_helpers.tpl:480:61: executing "confluence.databaseEnvVars" at <.Values.database.type>: wrong type for value; expected string; got interface {}
```
That's why I has to go with nested condition